src/Classes/Utils/Constants.ts

```diff
@@ -131,6 +132,19 @@ export class Constants {
                 value: 20,
             },
         },
+        polygon: {
+            draw: {
+                lineColor: "#ffffff",
+                lineWidth: 0.5,
+                enable: false,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'enable' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)